server/application.py

```diff
@@ -24,6 +24,11 @@ def alldata():
     return json.dumps(stats_helper.join_all())
 

```

Comment:
There should only be one blank line between methods inside of a class according to PEP8

Reply:
Removed blank lines